templates/cf-runalerts.service.mustache

```diff
@@ -3,6 +3,7 @@ Description=CFEngine Enterprise SQL Alerts
 After=syslog.target
 ConditionPathExists={{{vars.sys.bindir}}}/runalerts.php
 ConditionFileIsExecutable={{{vars.sys.workdir}}}/httpd/php/bin/php
+ConditionPathIsDirectory={{{vars.sys.workdir}}}/httpd/php/runalerts-stamp
```

Comment:
Seems like we would want this service to fail with an error, not just "silently" as I understand from https://www.man7.org/linux/man-pages/man5/systemd.unit.5.html

```suggestion
AssertPathExists={{{vars.sys.bindir}}}/runalerts.php
AssertFileIsExecutable={{{vars.sys.workdir}}}/httpd/php/bin/php
AssertPathIsDirectory={{{vars.sys.workdir}}}/httpd/php/runalerts-stamp
```

Reply:
I'll create a ticket. We can do this after release.